src/components/Tags/Tags.vue

```diff
@@ -1,65 +1,88 @@
 <template>
   <div class="tags">
-      <Tag v-for="tag in internalTags"
-           :key="tag"
-           v-bind="{icon, disabled, outlined, hovered, elevated, flat, color}"
-      >{{ tag }}</Tag>
+    <i v-if="icon" class="pi tags__icon" :class="icon" />
+    <Tag flat>
+      {{ internalTags }}
+      <template v-if="tagAmountLimit">
+        <Tooltip  placement="bottom">
+          <template v-slot:trigger>
+            ...<span class="tags__moreTooltip">{{hiddenTagsNumber}} more</span>
+          </template>
+          <span class="tags__tooltipTag">{{ hiddenTags }}</span>
```

Comment:
```suggestion
          <p class="tags__tooltipTag">{{ hiddenTags }}</p>
```

Reply:
`<p>` adds an extra margin on top and bottom

<img width="760" alt="Screen Shot 2021-12-17 at 4 33 54 PM" src="https://user-images.githubusercontent.com/40467112/146610569-224ddb18-e75b-4ca6-8893-3e4a1400093d.png">

Reply:
that's unfortunate. okay nevermind.